pylint/config/config_initialization.py

```diff
@@ -82,6 +83,9 @@ def _config_initialization(
     args_list = _order_all_first(args_list, joined=True)
     parsed_args_list = linter._parse_command_line_configuration(args_list)
 
+    # save Runner.verbose to make this preprocessed option visible from other modules
+    linter.config.verbose = verbose_mode
```

Comment:
I don't understand why this is necessary. Is ore logic for this flawed?

Reply:
Currently the only places where it is known that `--verbose` was set is inside `Run.__init__()` and in 2 functions that are called directly from there (`_config_initialization` and `linter.generate_reports`). 

When I add additional configs, I also add verbose messages about loading and applying of this configs. Since applying per-file configs can't happen in global `Run.__init__()`, information about verbosity should be copied to some place where several calls of `linter._lint_file` and downstream methods can access it.